Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM node:0.12-onbuild
```

Comment:
For the record, doc about this: https://github.com/docker-library/docs/tree/master/node#image-variants

Reading that doc, it sounds like we should be using `node:<version>` instead. On that note, it's what your own Dockerfile says, any reason why you don't want to do that here too?